scopus-import/src/main/java/no/sikt/nva/scopus/conversion/PiaConnection.java

```diff
@@ -103,19 +106,18 @@ private static String createAuthorization(SecretsReader secretsReader, Environme
         return String.format(BASIC_AUTHORIZATION, Base64.getEncoder().encodeToString(loginPassword.getBytes()));
     }
 
-    private URI createCristinUriFromCristinOrganization(Affiliation affiliation) {
-        return nonNull(affiliation.getUnitIdentifier())
-                   ? constructUri(affiliation)
-                   : null;
+    private Optional<URI> createCristinUriFromCristinOrganization(Affiliation affiliation) {
+        return Optional.ofNullable(affiliation)
+            .filter(input -> nonNull(input.getUnitIdentifier()))
+            .map(this::constructUri);
     }
 
     private URI constructUri(Affiliation affiliation) {
-        return attempt(() -> new URIBuilder()
-                                 .setHost(cristinProxyHost)
-                                 .setPath(CRISTIN_ORGANIZATION_PATH + affiliation.getUnitIdentifier())
-                                 .setScheme(HTTPS_SCHEME)
-                                 .build())
-                   .orElseThrow();
```

Comment:
Seems like a hard way to do this, it also maybe is the cause of "HTTP_SCHEME" above